def test_break():\n    assert False, 'forced failure'
